src/main/java/Problem1.java

```diff
@@ -2,7 +2,25 @@
 
     // Do not change signature (function name, parameters)
     public static int binarySearch(int[] data, int target) {
-        // Add your solution
-        return -1; // Place holder
+        // Set and return idx as data array length
+        int idx = data.length - 1;
+
+        // For loop goes through the array, could also use a while loop
+        for (int i = 0; i < data.length; i++)
+        {
+            // Finds target
+            if(data[i] == target)
```

Comment:
-10: linear search instead of binary search

Reply:
using the wrong algorithm will cost you lots more points in future homework. when in doubt, check with the instructor to make sure.